markdown/chapters/meandeck/deckconstruction.md

```diff
@@ -0,0 +1,603 @@
+As the game progresses they start being less effective but can be used
+*pitch* to Force of Will meaning they are never truly dead cards.
+For non-control based lists this is almost always a four-of include 
+into the deck as it really helps push through a lot of matchups. In
+control lists it tends to not be included at all, instead being
+replace with alternative tools like Baleful Strix and Teferi.
+
+There are a couple of additional counter spell effects you can elect
+to run but these are normally personal preference choices. 
+{{Flusterstorm}} and Spell Pierce are both viable options with the
+former being good at protecting your combo and against opposing combo
+and the latter being better at protecting from permanent hate like
+Blood Moon or planeswalkers like {{Karn the Great Creator}}. These
+normally contend with the proactive interaction slots and are only
+ever featured as a one or two-of at most. Some will also elect
+to run some number of {{Pact of Negation}} however this will 
+be discussed as a pile card.
+
+Veil of Summer is an interesting interaction card. It can act both as
+proactive interaction by initiating a combo turn with it, but can also
+act as reactive interaction as part of a counter war or defending against
+opposing discard effects or similar. We have elected to include it
+as reactive interaction because the most common usage is in a reactive sense
+over a proactive sense. It is very effective against opposing {U}
+or {B} based decks however it has almost no text in non-{U}
+matchups and thus is normally relegated to the sideboard. If it is run in
+the main it tends to replace some number of discard spells and is only
+run as a two to three-of.
+
+### 4.3 Deckbuilding Ratios - Interaction
+
+When combining the two interaction types, most lists will have around 
+a total of eleven to thirteen interaction slots in them. These tend to
+be split into eight to ten reactive slots and anywhere between two and
+eight proactive slots depending on the style of the list being played.
+As mentioned, almost all lists will have at a minimum four copies
+of Force of Will and at least one Duress and one Thoughtseize. The rest
+becomes quite flexible. 
+
+Let's consider some example mana bases for different styles of play for the deck.
+We shall present an aggressive, Grixis {U}{B}{R} *Turbo* package, a more balanced
+straight {U}{B} package and an Uro style BUG {B}{U}{G} package.
+
+#### {U}{B}{R} *Turbo*
+
+- 4 Force of Will
+- 4 Daze
+- 2 Thoughtseize
+- 1 Duress
+
+A no-nonsense approach, this list would likely run a heavy number of free cycle
+effects and a low number of lands. As a result it is relying on being able
+to go fast and not need to worry about having a lot of interaction, only enough
+to try and push through an early combo. It would also likely be supplemented by
+a Pact of Negation too.
+
+#### {U}{B} *Balanced*
+
+- 4 Force of Will
+- 4 Daze
+- 2 Thoughtseize
+- 3 Duress
+- 1 Flusterstorm
+
```

Comment:
Maybe we should use 13 interaction spells in this example.
So we have both ends of the spectrum metioned above.
"When combining the two interaction types, most lists will have around 
a total of eleven to thirteen interaction slots in them"

Reply:
I think I like the variants listed. All were taken from recent lists.